Helpers/Helper.cs

```diff
@@ -39,6 +39,12 @@ public static partial class Helper
         /// </summary>
         public static Vector2 TwoValueMax(this Vector2 vector, Vector2 vector2) => new Vector2(Math.Max(vector.X, vector2.X), Math.Max(vector.Y, vector2.Y));
         public static Player Owner(this Projectile proj) => Main.player[proj.owner];
+        public static IEnumerable<Enum> GetFlags(this Enum input)
```

Comment:
This function seems a bit unclear -- what exactly is it doing? You should add documentation to this and possibly give a more clear name